app/campaigns/campaigns.service.js

```diff
@@ -0,0 +1,30 @@
+const CampaignDAO = require('./campaigns.db')
+
+const CampaignService = {
+  getCampaigns,
+  getActiveCampaigns,
+  getCompleteCampaigns
+}
+
+async function getCampaigns () {
+  const campaigns = await CampaignDAO.getCampaigns()
+  return {
+    campaigns: campaigns
```

Comment:
Since ES6, you can define objects that have the same key as a variable name like
```
{
  campaigns,
}
```
There's an eslint rule for that as well

Reply:
And I also like the rule of ending each line of an object definition with a comma, even the last one.
That way you can easily edit the object definition and move lines. And there's no need to modify existing lines.